controller/tokenizer.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Dict, List
+from spacy.tokens import DocBin
+from submodules.model.business_objects import record
+from submodules.model.models import Record, RecordTokenized
+
+
+def get_doc_bin_in_bytes(
+    project_id: str,
+    tokenizer: str,
+    record_item: Record,
+    text_attributes: List[str],
+    update_statistic: bool = False,
+) -> Dict[str, Any]:
+    doc_bin = DocBin()
+    attribute_names_ordered = []
+    for key in record_item.data:
+        if key in text_attributes:
+            doc = tokenizer(record_item.data[key])
+            doc_bin.add(doc)
+            attribute_names_ordered.append(key)
+            if update_statistic:
+                record.create_or_update_token_statistic(
+                    project_id, record_item.id, text_attributes[key], len(doc)
+                )
+
```

Comment:
i think we should add a commit for the update case since the function itself doesn't commit atm


```suggestion
if update_statistic:
    general.commit()
```

Reply:
or maybe a different approach to ensure the values are committed to the db for the update case (single record tokenization if I found it correct)

Reply:
Added commit